src/app_utils/entity/__init__.py

```diff
@@ -22,3 +22,24 @@ def __get_entity(entity_uuid, auth_header=None):
         raise HTTPException(err_msg, response.status_code)
 
     return response.json()
+
+
+def get_entity_type_instanceof(type_a, type_b, auth_header=None) -> bool:
```

Comment:
@ChuckKollar I think @shirey has been promoting to use the python-sdk to handle entity-api calls inside ingest-api. Since you've added two new entity-api endpoints
-  `/entities/type/{type_a}/instanceof/{type_b}` and the 
- `/entities/<id>/instanceof/<type>`,

might be a better idea to add them to the https://github.com/hubmapconsortium/python-sdk then use the sdk call in ingest-api.

Reply:
From Bill:
>Yes, we should to include these there.  And consume using the SDK in ingest-api code.  We should do that in a separate branch/version from my recent additions to the SDK, though, as I don't want this stuff holding up releasing what I just did.